java/src/jmri/jmrit/roster/Roster.java

```diff
@@ -1232,6 +1233,9 @@ public void copyRosterGroupList(String oldName, String newName) {
             re.putAttribute(newGroup, "yes"); // NOI18N
         });
         this.addRosterGroup(new RosterGroup(newName));
+        // would expect the firePropertyChange event to be called by addRosterGroup()
+        // but has to be repeated here for the Group list to redraw:
+        firePropertyChange(ROSTER_GROUP_ADDED, null, newName);
```

Comment:
This is a bug for sure, but a bit too advanced maybe for me to fix.
I looked at the listener (in line 491 of roster.swing.RosterGroupsPanel) but just don't understand why it does work when a new group is created, but fails when being invoked after duplication. In both cases propertyName = "RosterGroupAdded". jmrit.roster.Roster line 1237 calls addRosterGroup() in line 1169 (String) which in turn calls line 1150 (rg). In fact the duplicate is added to the table (check by opening in a new window) but a debug shows it's not via this path and no change whatsoever is sent out. So there's no extra traffic and I hope we can solve it this way.